src/Vendor/Order/OrderListFilter.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * @author      Wizacha DevTeam <dev@wizacha.com>
+ * @copyright   Copyright (c) Wizacha
+ * @license     Proprietary
+ */
+declare(strict_types=1);
+
+namespace Wizaplace\SDK\Vendor\Order;
+
+use Wizaplace\SDK\ArrayableInterface;
+
+final class OrderListFilter implements ArrayableInterface
+{
+    /**
+     * @var int[]
+     */
+    private $companyIds = [];
+
+    /**
+     * @var string
+     */
+    private $lastStatusChangeIsAfter;
+
+    /**
+     * @var string
+     */
+    private $lastStatusChangeIsBefore;
+
+    /**
+     * @param int[] $companyIds
+     *
+     * @return OrderListFilter
+     */
+    public function byCompanyIds(array $companyIds): self
+    {
+        $this->companyIds = $companyIds;
+
+        return $this;
+    }
+
+    /**
+     * @param string $lastStatusChangeIsAfter
+     *
+     * @return OrderListFilter
+     */
+    public function byLastStatusChangeIsAfter(string $lastStatusChangeIsAfter): self
```

Comment:
Comme on disait toute à l'heure, c'est mieux d'utiliser un objet, comme ça l'utilisateur du sdk n'a pas a s'occuper du format qui vas être envoyé à l'API.
```suggestion
    public function byLastStatusChangeIsAfter(DateTime $lastStatusChangeIsAfter): self
```